lang/cy.json

```diff
@@ -594,7 +594,7 @@
     "whatHappensWhenYouSignTheLpaContentTrustCorporationPw": "<p class=\"govuk-body\">Welsh</p> <p class=\"govuk-body\">Welsh {{ .DonorFullNamePossessive }} welsh</p> <p class=\"govuk-body\">Welsh:</p>",
     "whatHappensWhenYouSignTheLpaContentTrustCorporationPa": "<p class=\"govuk-body\">Mae llofnodi’r atwrneiaeth arhosol yn gam cyfreithiol pwysig a phwerus.</p><p class=\"govuk-body\">Drwy lofnodi, rydych yn dweud yn swyddogol eich bod eisiau i {{.TrustCorporationName}} fod yn atwrnai ar atwrneiaeth arhosol eiddo a materion ariannol {{ .DonorFullNamePossessive }}.</p> <p class=\"govuk-body\">Mae hyn yn golygu fod gan {{.TrustCorporationName}} (ac unrhyw atwrneiod eraill a enwir yn yr atwrneiaeth arhosol hon) y pŵer i wneud y canlynol, unwaith y bydd yr atwrneiaeth arhosol wedi ei chofrestru:</p>",
     "whatHappensWhenYouSignTheLpaContentReplacementTrustCorporationPw": "<p class=\"govuk-body\">Welsh</p> <p class=\"govuk-body\">Welsh {{ .DonorFullNamePossessive }} welsh</p> <p class=\"govuk-body\">Welsh:</p>",
-    "whatHappensWhenYouSignTheLpaContentReplacementTrustCorporationPa": "<p class=\"govuk-body\">Welsh</p> <p class=\"govuk-body\">Welsh {{ .DonorFullNamePossessive }} welsh</p> <p class=\"govuk-body\">Welsh:</p>",
+    "whatHappensWhenYouSignTheLpaContentReplacementTrustCorporationPa": "<p class=\"govuk-body\">Mae llofnodi’r atwrneiaeth arhosol yn gam cyfreithiol pwysig a phwerus.</p>\n<p class=\"govuk-body\">Drwy lofnodi, rydych yn dweud yn swyddogol eich bod eisiau i {{.TrustCorporationName}} fod yn atwrnai wrth gefn ar atwrneiaeth arhosol eiddo a materion ariannol {{.DonorFullNamePossessive}}.</p>\n<p class=\"govuk-body\">Mae hyn yn golygu unwaith y bydd yr atwrneiaeth arhosol wedi’i chofrestru, os oes angen {{.TrustCorporationName}} gamu i mewn ar gyfer atwrnai gwreiddiol, bydd ganddo’r pŵer i:</p>\n",
```

Comment:
We also wouldn't want to include newline characters (`/n`) here as they may affect the layout of the page.